openstack/intel-worker/playbooks/roles/virtual_machine_create/templates/worker-amd64.ign.j2

```diff
@@ -0,0 +1,49 @@
+{
+	"ignition": {
+		"version": "3.4.0",
+		"config": {
+			"merge": [
+				{
+					"source": "http://{{ virtual_machine_create_bastion_ip }}:{{ virtual_machine_create_http_port }}/ignition/worker-amd64.ign"
+				}
+			]
+		}
+	},
+	"timeouts": {
+		"httpTotal": 15
+	},
+	"storage": {
+	  "files": [
+		{
+				"group": {},
+				"path": "/etc/hostname",
+				"user": {},
+				"contents": {
+					"source": "data:text/plain;base64,{{ virtual_machine_create_worker_hostname_encoded }}",
+					"verification": {}
+				},
+				"mode": 420
+		},
+		{
+				"group": {},
+				"path": "/etc/resolv.conf",
+				"user": {},
+				"contents": {
+					"source": "data:text/plain;base64,{{ virtual_machine_create_etc_resolve_encoded }}",
```

Comment:
```suggestion
					"source": "data:text/plain;base64,{{ etc_resolve_encoded }}",
```

Reply:
Ansible lint expects to have a role name as a prefix for the variable names .